readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia'), required=False)
+
+# The path to the requirements file from the root of the project
+requirements_file: str(required=False)
+
+# Configuration for Conda support
+conda: include('conda', required=False)
+
+# Configuration for the documentation build process
+build: include('build', required=False)
+
+# Configuration of the Python executable to be used
+python: include('python', required=False)
+
+# Configuration for sphinx
+sphinx: include('sphinx', required=False)
+
+# Submodules configuration
+submodules: include('submodules', required=False)
+
+# Redirects for the current version to be built
+redirects: include('redirects', required=False)
+
+---
+
+conda:
+  # The path to the Conda environment file from the root of the project
+  file: str()
+
+build:
+  # The build docker image to be used
+  # Default: '2.0'
+  image: enum('1.0', '2.0', 'latest')
+
+python:
+  # The Python version
+  # Default: '2.7'
+  version: enum('2', '2.7', '3', '3.3', '3.4', '3.5', '3.6', required=False)
+
+  # Install the project using python setup.py install or pip
+  install: enum('pip', 'setup.py', required=False)
+
+  # Extra requirements sections to install in addition to the package dependencies
+  # Default: []
+  extra_requirements: list(str(), required=False)
+
+  # Give the virtual environment access to the global site-packages dir
+  # Default: false
+  system_packages: bool(required=False)
```

Comment:
I wasn't 100% sure if this should belong in the `python` key

Reply:
I mostly want this option to go away, but I don't know if we can do it quite yet. I think it _can_ go in `python` though.